openapi/ghc.yaml

```
@@ -0,0 +1,401 @@
+openapi: 3.0.2
+info:
+  title: move.mil GHC API
+  description: The API for the GHC Prime to interact with move.mil
+  #  termsOfService:
+  contact:
+    email: dp3@truss.works
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+  version: 0.0.1
+# servers:
+paths:
+  /payment_requests:
+    post:
+      summary: Creates a payment request
+      description: Creates a payment request
+      operationId: createPaymentRequest
+      tags:
+        - payment_requests
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreatePaymentRequestPayload'
+      responses:
+        '201':
+          description: created instance of payment request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PaymentRequest'
+        '400':
```

I think this can be shortened to just a reference:

```suggestion
        '400':
          $ref: '#/components/responses/InvalidRequestResponsePayload'
```

I think it's worth defining more shared error response types that we can reference; I took a stab at adding some [in this commit](https://github.com/transcom/mymove/commit/68de9fa5c415f9cbd753124f883ddd493956e071).